langchain/src/chains/question_answering/refine_prompts.ts

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable tree-shaking/no-side-effects-in-initialization */
+/* eslint-disable spaced-comment */
+import {
+  PromptTemplate,
+  ChatPromptTemplate,
+  SystemMessagePromptTemplate,
+  HumanMessagePromptTemplate,
+  AIMessagePromptTemplate,
+} from "../../prompts/index.js";
+import { ConditionalPromptSelector, isChatModel } from "../prompt_selector.js";
+
+export const DEFAULT_REFINE_PROMPT_TMPL = `The original question is as follows: {question}
+    We have provided an existing answer: {existing_answer}
```

Comment:
it looks like theres some rouge tabs here?

Reply:
Fixed this.